### ISSUE TITLE
Syntax Error (Screenshots Attached)

### ISSUE THREAD

**Issue by jerrygviji** [mclmza/linkedin-learning-downloader#31]
![vhvhh](https://user-images.githubusercontent.com/37071082/36974121-8e0fc5ce-209b-11e8-94f4-f04a1f148f4e.png)

![sdsdsd](https://user-images.githubusercontent.com/37071082/36974128-928c28f4-209b-11e8-8627-5d24f4959017.png)

![sdsfd](https://user-images.githubusercontent.com/37071082/36974134-9454e1a8-209b-11e8-859e-705ea7e068ec.png)

**for testing, I tried to download the default course itself

**Comment by mclmza** [comment]
You must use Python 2.7

**Comment by jerrygviji** [comment]
yes, but now showing "ImportError: No module named requests"

pls check new 3 screenshots

**Comment by johnhern2017** [comment]
You first need to install the actual Requirements.txt file.. 
Then it will not give you that error.
E.g. pip install -r c:\users\username\desktop\linkedinvideostuff\requirements.txt

**Comment by jerrygviji** [comment]
D:\linkedin>python lld.py
[Fatal] Your IP may have been temporarily blocked
[Notice] Exception hit



Now, am getting this error

**Comment by johnhern2017** [comment]
Try and see if you can load the URL to that exact video in a regular browser.

**Comment by jerrygviji** [comment]
Tried to download this course: https://www.linkedin.com/learning/it-security-foundations-core-concepts/welcome

here is the config. file screenshot

![jhjjjj](https://user-images.githubusercontent.com/37071082/36983206-c82822da-20b7-11e8-8617-7b352cf63994.png)

-> Am using a trial account
-> Able to access that course directly through browser

but, showing that same error

**Comment by johnhern2017** [comment]
Hmm, 

Try commenting that one out. Completely. 
As well as any others.. and then add a new one... just randomly right click on a "course", and then copy and paste that link and delete everything from https:\\whatever.com\learning\this-is-what-you-want-to-leave... 

then save it and try it again. 

Are you in windows/mac/linux?

**Comment by jerrygviji** [comment]
Windows

**Comment by jerrygviji** [comment]
Not working

**Comment by johnhern2017** [comment]
Just for fun, can you take out the comma after the 'whatever-course' if it is the only one in there. 
COURSES = [
    # 'it-security-foundations-core-concepts',
    # 'charlene-li-on-digital-leadership',
    # 'fred-kofman-on-managing-conflict',
    # 'reid-hoffman-and-chris-yeh-on-creating-an-alliance-with-employees',
    # 'bill-george-on-self-awareness-authenticity-and-leadership',
    # 'sallie-krawcheck-on-risk-taking',
    # 'betsy-corcoran-on-choosing-the-right-technology-for-your-school',
    # 'betty-liu-on-career-success',
    # 'j-t-o-donnell-on-making-recruiters-come-to-you',
    # 'gary-hamel-on-busting-bureaucracy',
    # 'shane-snow-on-storytelling',
    # 'john-maeda-on-design-business-and-inclusion',
    # 'esther-dyson-on-cultivating-health-at-scale',
    # 'influencer-interview-oprah-winfrey'
	# 'excel-2016-esencial'
	# 'excel-2016-analisis-y-gestion-de-datos'
	# 'excel-2016-tablas-dinamicas'
	# 'excel-2016-formulas-y-funciones'
	 'excel-2016-graficos-y-presentacion-de-datos'
	# 'excel-2016-validacion-de-datos'
	# 'excel-2016-trucos'
]

I am doing them one at a time, and taking out that comma at the end.

**Comment by Miykoll** [comment]
Don't know if your problem still exist, but I had the same problem.
My solution was to create a virtualenv with python 2.7, activate it and then do the pip install and run the script.

Since I got Python 2.7 and Python 3 on Ubuntu 16.04, I think pip uses pip3. I ran pip install and it only installed beautifulsoup4. Running the installer again in the virtualenv installs me a few more modules, including the requests module. Tested the script and successfully downloaded the first course in the demo config.py file.

See here the terminal results when installing in virtualenv:
![image](https://user-images.githubusercontent.com/9033311/37925920-25ab856c-3136-11e8-89be-916d14f4fad0.png)
